astropy/cosmology/tests/test_cosmology.py

```diff
@@ -104,7 +104,8 @@ def test_units():
     """ Test if the right units are being returned"""
 
     cosmo = core.FlatLambdaCDM(H0=70, Om0=0.27, Tcmb0=2.0)
-    assert cosmo.comoving_distance(1.0).unit == u.Mpc
+    with pytest.warns(RuntimeWarning, match='numpy.dtype size changed'):
+        assert cosmo.comoving_distance(1.0).unit == u.Mpc
```

Comment:
I'm curious where this warning comes from - that's normally the kind of warning that happens when numpy has some installation issues... Do you understand why the warning happens?

Reply:
I don't see the warning locally, so part of me thinks this is an issue specific to your installation (maybe scipy was compiled with an older version of numpy). I think this warning check should be removed.

Reply:
> Do you understand why the warning happens?

I don't know. I probably did something unholy in my conda environment.